src/Microsoft.ML.ImageAnalytics/ImageResizer.cs

```diff
@@ -135,7 +135,7 @@ private static VersionInfo GetVersionInfo()
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="imageWidth">Width of resized image.</param>
         /// <param name="imageHeight">Height of resized image.</param>
-        /// <param name="inputColumnName">Name of the input column.</param>
+        /// <param name="inputColumnName">The name of the input column.</param>
```

Comment:
>inputColumnName [](start = 25, length = 15)

 /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>